apps/fetch_app_images.py

```diff
@@ -0,0 +1,41 @@
+import argparse
+import logging
+import os
+import subprocess
+
+from apps.compose_apps import ComposeApps
+from apps.apps_publisher import AppsPublisher
+from helpers import status
+
+
+def main(args):
+    status("Fetching app images to the local docker engine store")
+
+    publisher = AppsPublisher(args.factory, "", "")
+    apps = ComposeApps(args.apps_root, quiet=True)
+    publisher.tag(apps, args.tag)
```

Comment:
so the trick here and why we need all this python vs a "docker compose pull"  is because of the tagging complexities, right?

Reply:
Yes. In addition to that we need to skip the disabled apps and the "image directories" (directories that doesn't have docker-compose.yml).